src/user/importexport-app.md

```diff
@@ -399,56 +400,90 @@ You can export event or tracker data in JSON, CSV, or XML formats.
+
+
+    - _Accessible_: to select data view organisation units associated with the current user
+
+    - _Capture_: to select data view organisation units associated with the current user.
+
+    - _Manually select organisation units_: to manually select the organisation units.
 
-1.  Select the organisation units that should be included.
+1. If you choose to _manually select organisation units_ then you will be presented with further options:
 
-2.  Decide whether you want to filter by program or tracked entity type.
+    - _Selected_: Export data only for the selected
+      organisation unit
+
+    - _Directly below_: Export data including the first
+      level of the organisation units inside the selections as well
+      as the selected organisation unit itself.
+
+    - _All below_: Export event data for all organisation units
+      inside the selections as well as the selected organisation
+      unit itself.
+
+1.  Decide whether you want to filter by _program_ or _tracked entity type_.
 
-3.  Select a format: _JSON_, _CSV_, or _XML_.
+1. Decide what statuses to include in the export
+
+1. Decide which follow-up statuses to include in the export
+
+3.  Select a format: _JSON_ or _CSV_.
 
 4.  Click **Advanced options** if you want to adjust one or more of
     the following settings before exporting:
 
     - Filter by last updated date
-    - Assigned user mode
+    - Filter by assigned user
     - Include deleted
-    - Include all attributes
     - Data element ID scheme
     - Event ID scheme
     - Organisation unit ID scheme
     - ID scheme
 
-5.  Click **Export tracked entity instances** which will open a new
+5.  Click **Export tracked entity** which will open a new
     web-browser window that will give you a file to download to your
     local computer.
 
+## Changes to Tracker import/export in v41+ { #v41_tracker_changes }
+
+The import/export app was upgraded to use the new tracker API for importing and exporting tracked entities and events. The old API will be deprecated in v42 hence all applications are advised to upgrade as soon as possible.
+
+This brings a lot of benefits in terms of the consistency and reliability of the import/export experience for tracked entities and events, such as better validation, error reporting and the use of a more reliable job scheduling workflow.
+
+These benefits come with a couple of important caveats. First, the new format of the exported files is incompatible with previous versions of the app, so exports from v41 can not be imported in previous versions, and exports from previous versions can not be imported in v41. We also dropped support for XML format, and only support JSON and CSV. More context for these changes can be found in the [Tracker API documentation](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html).
+
+![Improved error reports in v41+](resources/images/import_export/v41_error_reports.png)
+
+
```

Comment:
@teleivo @Bekkalizer this bit here specifically is that I would like a second look from you about - does it sound ok? is there anything more that I should add about the tracker upgrade that's relevant from an end-user perspective